services/scan/src/importer.ts

```diff
@@ -563,9 +563,8 @@ export class Importer {
   /**
    * Reset all the data, both in the store and the ballot images.
    */
-  async doZero(): Promise<void> {
+  doZero(): void {
     this.workspace.zero();
-    await this.setMarkThresholdOverrides(undefined);
```

Comment:
I'm not sure why we're resetting the mark threshold overrides in `doZero`. We shouldn't reset the thresholds when we switch between test modes, or clear ballot data. On unconfigure they will be cleared anyway. Removed.